lib/new_relic/agent/configuration/security_policy_source.rb

```diff
@@ -35,6 +35,8 @@ def change_setting(policies, option, new_value)
           end
         end
 
+        # The Language Security Policy Source gives customers the ability to
+        # configure high security mode settings.
```

Comment:
Two small, totally optional, suggestions:
1. If you move this comment between lines 9 and 10, just before the class declaration, it'll get added to RubyDoc's YARD documentation for the class. That's where I expect to see comments like this.

2. One of the things I found most helpful about your description of this class was its dubious context. I recommend adding something like: 
"This source never made it to general availability (no public docs) and work on it has been abandoned. We're uncertain if any of our customers use this."
This may help future engineers more quickly know how to prioritize work in this class.